commands/joined.js

```diff
@@ -13,30 +12,26 @@ module.exports = {
   description: 'See when someone joined the server.',
   allowDM: false,
   process: (bot, message) => {
-    message.mentions.users.forEach(user => {
-      const member = message.guild.member(user);
-      let joined = moment(member.joinedAt);
-
-      if (user.id === REYNBOW.id) {
-        joined = REYNBOW.joinedAt;
-      }
-
-      const embed = new Discord.RichEmbed();
-
-      embed.setColor(0x3398DB);
-      embed.setTitle(member.displayName + ' joined ' + moment(joined).fromNow());
-
-      if (user.id === REYNBOW.id) {
-        embed.setTitle(member.displayName.toUpperCase() +
-          ' BOUNCED IN HERE LIKE A FEISTY \'ROO ABOUT ' +
-          moment(joined).fromNow().toUpperCase() + ', MATE.');
-      }
-
-      embed.addField('User', member);
-
-      embed.setTimestamp(joined);
-
-      message.channel.sendMessage('', { embed: embed });
-    });
+    let target = message.guild.member(message.mentions.users.first());
+    if (!target) {
+      target = message.member;
+    }
+
+    const embed = new Discord.RichEmbed();
+    embed.setColor(0x3398DB);
+    embed.addField('User', target);
+
+    if (target.id === REYNBOW.id) {
+      embed.setTitle(target.displayName.toUpperCase() +
+        ' BOUNCED IN HERE LIKE A FEISTY \'ROO ABOUT ' +
+        moment(REYNBOW.joinedAt).fromNow().toUpperCase() + ', MATE.');
+      embed.setTimestamp(REYNBOW.joinedAt);
+    } else {
+      embed.setTitle(target.displayName + ' joined ' +
+        moment(target.joinedAt).fromNow());
+      embed.setTimestamp(target.joinedAt);
+    }
+
+    message.channel.sendMessage('', { embed: embed });
```

Comment:
Don't know if it matters much as these aren't critical messages, but the empty string here should have something in it if you're wanting the post to be searchable.

Approving anyway though as I don't think its vital.